symplyphysics/laws/astronomy/luminosity_of_Sun_in_future_from_luminosity_in_present.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve, S,)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The luminosity of the Sun in the future is related to the luminosity of the Sun in the present. Luminosity is
+## indicated in units of solar luminosity. One unit is equal to the luminosity of the Sun at a given time.
+
+## Law is: Lf = L * ((5.59 / t) - 1.41 + 0.26 * t), where
+## Lf - luminosity of the sun in future,
+## L - luminosity of the sun in present,
+## t - time in billions of years.
+
+luminosity_future = Symbol("luminosity_future", dimensionless)
+
+luminosity_present = Symbol("luminosity_present", dimensionless)
+time = Symbol("time", dimensionless)
+
+law = Eq(luminosity_future, luminosity_present * ((5.59 / time) - 1.41 + 0.26 * time))
```

Comment:
Why don't you divide time to 1 billion years to allow passing any sort of time periods?